tests/integration/conftest.py

```diff
@@ -777,19 +873,16 @@ def yum_conf_exclude(shell, backup_directory, request):
 
         def test_function(yum_conf_exclude):
     """
-    # We need to do this only for Oracle Linux
-    # rhn* is one of the excluded packages in convert2rhel configs
-    # so if the package is present on the system, is excluded in yum.conf
-    # and convert2rhel tries to back it up by calling yumdownloader
-    # it fails to do so because the call conflicts with the exclude option
-    if "oracle" in SYSTEM_RELEASE_ENV:
-        exclude = ["rhn-client*"]
-        if hasattr(request, "param"):
-            exclude = request.param
-            print(">>> Using parametrized packages requested in the fixture.")
-        yum_config = "/etc/yum.conf"
-        backup_dir = os.path.join(backup_directory, "yumconf")
-        shell(f"mkdir -v {backup_dir}")
+    exclude = [""]
+    if hasattr(request, "param"):
+        exclude = request.param
+        print(">>> Using parametrized packages requested in the fixture.")
+    yum_configs_all = ["/etc/yum.conf", "/etc/dnf/dnf.conf"]
+    # work only with existing config files
```

Comment:
I heard that the `/etc/dnf/dnf.conf` is not needed. You can write to the /etc/yum.conf and it gets redirected to dnf.conf if applicable. But I would comment this behavior if we use it.

Reply:
Yeah, on systems with dnf as package manager, `/etc/yum.conf` is just a symlink with `/etc/dnf/dnf.conf` as a target.
I'm not sure if `/etc/dnf/dnf.conf` is or is not needed, but I would rather keep it, if it exists on the system in the first place.
This was supposed to be more or less generalized implementation which would also ensure a proper restore of all configs in their vanilla settings.

Reply:
We can refactor this further in some future iterations, I feel like it's out of scope of this PR which is already bigger than necessary.
We plan to rework the whole `conftest.py` anyway

Reply:
yes, sounds nice